sdks/js/src/types.ts

```diff
@@ -1342,9 +1350,24 @@ const NotificationTextContentSchema = z.object({
   text: z.string(),
 });
 
+const ToolApproveExecutionSchema = z.object({
+  type: z.string(),
+  configurationId: z.string(),
+  conversationId: z.string(),
+  messageId: z.string(),
+  actionId: z.string(),
+  inputs: z.record(z.any()),
+  stake: MCPStakeLevelSchema,
+  metadata: MCPValidationMetadataSchema,
+});
+
 const NotificationContentSchema = z.union([
   NotificationImageContentSchema,
   NotificationTextContentSchema,
+  z.object({
```

Comment:
Let's maintain consistency and hav NotificationToolApproveExecutionSchema here and define it above

Reply:
oops sorry forgot to push this fix